modules/bliinkBidAdapter.js

```diff
@@ -156,61 +137,58 @@ export const buildBid = (bidRequest, bliinkCreative) => {
  * @return boolean
  */
 export const isBidRequestValid = (bid) => {
-  return !(!bid || !bid.params || !bid.params.placement || !bid.params.tagId)
-}
+  return !!(bid && bid.params && !!bid.params.tagId);
+};
 
 /**
  * @description Takes an array of valid bid requests, all of which are guaranteed to have passed the isBidRequestValid() test.
  *
- * @param _[]
+ * @param validBidRequests
  * @param bidderRequest
- * @return {{ method: string, url: string } | null}
+ * @returns {null|{method: string, data: {gdprConsent: string, keywords: string, pageTitle: string, pageDescription: (*|string), pageUrl, gdpr: boolean, tags: *}, url: string}}
  */
-export const buildRequests = (_, bidderRequest) => {
-  if (!bidderRequest) return null
+export const buildRequests = (validBidRequests, bidderRequest) => {
+  if (!validBidRequests || !bidderRequest || !bidderRequest.bids) return null
 
-  let data = {
+  const tags = bidderRequest.bids.map((bid) => {
+    return {
+      sizes: bid.sizes.map((size) => ({ w: size[0], h: size[1] })),
+      id: bid.params.tagId,
+      transactionId: bid.bidId,
+      mediaTypes: Object.keys(bid.mediaTypes),
+      imageUrl: bid.params.imageUrl || '',
```

Comment:
this seems to be a new possible param, could this be added to your bidder markdown file and a PR opened for the prebid site repo to update your bidder information there as well. It is missing the update from params.placement to params.tagId